File: atv2/atv2.c
#include<stdio.h>

int main() {
    int horas,horasTotal,horasFim,validacao;
    printf("Digite quantas horas por dia você estuada: \n");
    scanf("%d",&horas);
    printf("você estuda final de semana? \n 1.sim 2.não");
    scanf("%d",&validacao);
    if(validacao ==1){
        printf("Digite quantas horas você estuda por final de semana: \n");
        scanf("%d",&horasFim);
        horasTotal = horas *22 + horasFim * 8;
        printf("voce estuda por mês: %d",horasTotal);
    }
    else{
        horasTotal = horas *22;
        printf("voce estuda por mês: %d",horasTotal);
    }
}

File: atv2/atv2.cs
using System;

class Program
{
    static void Main() {
        string horasE,horasTotalE,horasFimE,validacaoE;
        int horas,horasTotal,horasFim,validacao;
        Console.WriteLine("Digite quantas horas você estuda por semana: \n");
        horasE = Console.ReadLine();
        horas = int.Parse(horasE);
        Console.WriteLine("Você estuda final de semana? 1)sim 2)não \n");
        validacaoE = Console.ReadLine();
        validacao = int.Parse(validacaoE);
        if(validacao == 1){
            Console.WriteLine("Digite quantas horas estuda por final de semana: \n");
            horasFimE = Console.ReadLine();
            horasFim = int.Parse(horasFimE);
            horasTotal = horas * 22 + horasFim * 8;
            Console.WriteLine(horasTotal);
        }
        else{
           horasTotal = horas * 22;
            Console.WriteLine(horasTotal);
        }
       
    }
}
